### DISCUSSION TITLE
Wake-on-Lan possible?

### DISCUSSION THREAD

**Discussion by ghost2289** [bb-qq/r8152#409] [Q&A] [answered]
Is it possible to have WOL functionality?
Everything else seems good with my UGREEN usb-c 2023, DS423+, rear usb port.
Thanks

**Comment by bb-qq** [comment, marked as answer]
The Ethernet dongle and driver support WoL, but Synology side does not seem to support it.
If you need WoL, please use the built-in port together. There should be no need to assign an IP address if you only use MagicPacket.

**Reply by bb-qq** [reply]
I think I have answered your question, so I will close this thread. If you have any questions, feel free to re-open it.
